dev/com.ibm.ws.beanvalidation.v20_fat/.classpath

```diff
@@ -9,7 +9,12 @@
 	<classpathentry kind="src" path="test-applications/MultipleValidationXmlEjb2.jar/src"/>
 	<classpathentry kind="src" path="test-applications/MultipleValidationXmlWeb.war/src"/>
 	<classpathentry kind="src" path="test-applications/HibernateConfig.war/src"/>
+	<classpathentry kind="src" path="test-applications/bvalELApp/src"/>
```

Comment:
This is adding a new test application (bvalELApp) to the src path in eclipse so it will be built.